lib/packages/docutils/rst.nim

```diff
@@ -626,10 +671,14 @@ proc isInlineMarkupEnd(p: RstParser, markup: string): bool =
     if markup != "``" and prevTok(p).symbol == "\\":
       result = false
 
-proc isInlineMarkupStart(p: RstParser, markup: string): bool =
+proc isInlineMarkupStart(p: RstParser, markup: string, twoTokens=false): bool =
```

Comment:
not really, because other passed markup arguments include 1,2,3 characters, e.g. *, **, ***. But because this _ \` is an only case of 2 tokens, it's reasonable to get rid of `twoTokens` by hard-coding condition `markup == ..`